sponsorship/build.gradle

```diff
@@ -56,6 +58,23 @@ dependencies {
     testCompile group: 'org.mockito', name: 'mockito-junit-jupiter', version: '2.17.0'
     compile group: 'com.github.database-rider', name: 'rider-core', version: '1.7.2'
     compile group: 'com.github.database-rider', name: 'rider-junit5', version: '1.7.2'
+    // https://mvnrepository.com/artifact/org.apache.kafka/kafka
```

Comment:
this comments are redundant, it is clear from 'group' and 'name'

Reply:
Sorry, it was added automatically, removed